### DISCUSSION TITLE
Spacing between icon and label does not scale with display scaling

### DISCUSSION THREAD

**Discussion by mattcoz** [valinet/ExplorerPatcher#534] [Q&A]
I just got a new W11 laptop with a 4K screen set to 200% scaling. I installed this patcher and it works great, but there is very little spacing between the icon and label. It seems that the spacing does not scale along with the display scaling setting. I even tried setting it all the way to 400% and the spacing remained the same. I've compared it with my W10 laptop and the spacing does scale. An issue with the patcher or just an issue with using the W10 taskbar on W11?

**Comment by BraINstinct0** [comment]
Could you post an image that shows this issue? I think I got what you meant but I'm not totally sure. :)

**Comment by valinet** [comment]
It’s probably an issue with the W10 taskbar from W11’s explorer. I just enable that and do any fixes I can do on it. I never looked this, so I don’t touch it, so it’s untouched from Microsoft, and broken that is, if you say it’s broken.

**Reply by mattcoz** [reply]
Ok, figured that might be the case, thanks.
